drivers/sensor/lps22hh/Kconfig

```diff
@@ -72,4 +72,18 @@ config LPS22HH_SAMPLING_RATE
 
 endmenu
 
+config LPS22HH_LPF_BADWIDTH
+	int "Low-pass filter bandwidth"
+	range 0 3
+	default 0
+	help
+	  Low-pass filter configurations
+	  0: ODR/2 (LPF Disabled)
+	  1: N/A
+	  2: ODR/9
+	  3: ODR/20
+
+config LPS22HH_LOW_NOISE
+	bool "Low noise mode"
+
```

Comment:
What do you think of putting the LPF option in the DT instead of here in Kconfig?
(zephyr/dts/bindings/sensor/)

I suggest you to create a common dts file for lps22hh that can be shared among i2c and spi.

Reply:
Yes, it definitely needs to be done in DT. I will do this as soon as I will have more free time.